plugin-server/src/utils/db/db.ts

```diff
@@ -418,6 +422,13 @@ export class DB {
         }
     }
 
+    private fixFetchedPersonCreatedAt(selectResult: any) {
```

Comment:
context?

Reply:
`fetchPersonByPersonId` is used in tests